pkg/controller/petset/pet.go

```diff
@@ -297,22 +297,23 @@ type petHealthChecker interface {
 // It doesn't update, probe or get the pod.
 type defaultPetHealthChecker struct{}
 
-// isHealthy returns true if the pod is running and has the
-// "pod.alpha.kubernetes.io/initialized" set to "true".
+// isHealthy returns true if the pod is running and if it has the
+// "pod.alpha.kubernetes.io/initialized" annotation, it is set to "true".
 func (d *defaultPetHealthChecker) isHealthy(pod *api.Pod) bool {
 	if pod == nil || pod.Status.Phase != api.PodRunning {
 		return false
 	}
 	initialized, ok := pod.Annotations[PetSetInitAnnotation]
```

Comment:
nit might as well rename the variable holding the annotation to StatefulSetInitAnnotation or whatever we're going to use as the suffix